app/views/meeting.status.html

```diff
@@ -3,21 +3,53 @@
-                <i ng-if="menuFull()" ng-if="!msc.mobile" class="col-sm-3 glyphicon glyphicon-home left" aria-hidden="true" ng-click="msc.goHome()"></i>
-                <i ng-if="!msc.mobile" class="col-sm-3 glyphicon glyphicon-menu-left left" aria-hidden="true" ng-click="closeMenu()"></i>
-                <i ng-if="!msc.mobile" class="col-sm-3 glyphicon glyphicon-menu-right left" aria-hidden="true" ng-click="openMenu()"></i>
+                <i ng-if="menuFull()" class="col-xs-1 col-sm-3 col-md-2 col-lg-3 glyphicon glyphicon-info-sign left" aria-hidden="true" ng-click="msc.showInfo()"></i>
+                <i ng-if="menuFull()" class="col-sm-3 col-md-2 col-lg-3 glyphicon glyphicon-home left" aria-hidden="true" ng-click="msc.goHome()"></i>
+                <i class="col-sm-3 col-md-2 col-lg-3 xs-hidden glyphicon glyphicon-menu-left left" aria-hidden="true" ng-click="closeMenu()"></i>
+                <i class="col-sm-3 col-md-2 col-lg-3 xs-hidden glyphicon glyphicon-menu-right left" aria-hidden="true" ng-click="openMenu()"></i>
             </div>
         </div>
     </div>
 
     <div class="ad-page-content">
 
-        <button type="button" class="ng-class: ['btn', 'btn-primary', 'btn-lg']" ng-click="">{{msc.object.meetingTitle}} <span class="badge">8</span>
-        </button>
-        <button type="button" class="btn btn-primary btn-lg" ng-click="">Kokousstatus <span class="badge">9</span>
-        </button>
+        <button type="button" class="btn btn-primary btn-lg disabled ad-dummy-button" ng-click="">{{msc.object.meetingTitle}}</button>
+
+        <div class="btn-group btn-block" uib-dropdown is-open="">
+            <button id="statusButton" type="button" class="btn btn-primary btn-lg ad-button" uib-dropdown-toggle>
+                <div>{{msc.object.meetingStatus}} {{msc.object.topicList.length}} Status</div>
+                <span class="caret"></span>
+            </button>
+            <ul uib-dropdown-menu role="menu" aria-labelledby="statusButton">
+                <li role="menuitem" ng-click=""><a>First</a></li>
+                <li role="menuitem" ng-click=""><a>Second</a></li>
+            </ul>
+        </div>
+
+
+
+        <div class="ad-topic" ng-repeat="item in msc.object.topicList track by $index">
+
+            <div class="btn-group ad-topic-status" uib-dropdown is-open="" ng-init="status = item.Status">
+                <button id="topicButton" type="button" class="btn btn-default btn-lg ad-button" uib-dropdown-toggle>
+                    <img ng-if="status===1" src="images/state_4.png"></img>
+                    <img ng-if="status===2" src="images/state_4_lock.png"></img>
+                    <img ng-if="status===3" src="images/state_4_lock_remark.png"></img>
+                    <img ng-if="status===4" src="images/state_4_remark.png"></img>
+                    <span class="caret"></span>
+                </button>
+                <ul uib-dropdown-menu role="menu" aria-labelledby="topicButton">
```

Comment:
Let's not implement the status changing dropdowns at this phase, only later when extended to cover the specific user role.